BackBLAST.sh

```diff
@@ -0,0 +1,459 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# BackBLAST.sh
+# The entry script for running BackBLAST via command line
+# Copyright Lee H. Bergstrand and Jackson M. Tsuji, 2019
+
+
+# HARD-CODED variables
+VERSION=1.9.9-alpha
+SCRIPT_NAME=${0##*/}
+SCRIPT_NAME=${SCRIPT_NAME%.*}
+TEMPLATE_CONFIG="template_config.yaml"
+SNAKEFILE=Snakefile
```

Comment:
Longer-term, we'll need to figure out how to specify where these key files are in relation to the shell script (or just have them install to the same folder as the script?)

Reply:
Good idea -- done